TypeScript.py

```diff
@@ -986,6 +1030,169 @@ def on_selected(self, index):
                               {"contents": self.snippets[index]})
 
 
+class TypescriptSignaturePopup(sublime_plugin.TextCommand):
+    def is_enabled(self):
+        return is_typescript(self.view)
+
+    def run(self, edit, move=None):
+        print('In run for signature panel')
```

Comment:
as we already have a logging module, may be easier to control the output if replacing print with logger.log.debug or other level